### PR TITLE
feat(tests): Adds avatar test.

### PR DESCRIPTION
**Pull request by damencho**

### CODE REVIEW

File: tests/helpers/Participant.ts [hunk 1]
```diff
@@ -383,4 +404,87 @@ export class Participant {
...
     }
+
+    /**
+     * Gets avatar SRC attribute for the one displayed on local video thumbnail.
+     */
+    async getLocalVideoAvatar() {
+        const avatar
+            = this.driver.$('//span[@id="localVideoContainer"]//img[contains(@class,"userAvatar")]');
+
+        return await avatar.isExisting() ? await avatar.getAttribute('src') : null;
```
Comment (saghul) [inline tests/helpers/Participant.ts:415]:
```suggestion
        return await (avatar.isExisting() ? avatar.getAttribute('src') : null);
```

Comment (saghul) [reply tests/helpers/Participant.ts:415]: I think this would work.

File: tests/helpers/Participant.ts [hunk 2]
```diff
@@ -383,4 +404,87 @@ export class Participant {
...
+        const avatar = this.driver.$('//img[@id="dominantSpeakerAvatar"]');
+
+        return await avatar.isExisting() ? await avatar.getAttribute('src') : null;
+    }
+
+    /**
+     * Returns resource part of the JID of the user who is currently displayed in the large video area.
+     */
+    async getLargeVideoResource() {
+        return this.driver.execute(() => APP.UI.getLargeVideoID());
```
Comment (saghul) [inline tests/helpers/Participant.ts:431]: no await?

File: package.json [hunk 3]
```diff
@@ -182,6 +183,7 @@
     "jsonwebtoken": "9.0.2",
     "metro-react-native-babel-preset": "0.77.0",
     "patch-package": "6.4.7",
+    "pretty": "2.0.0",
```
Comment (saghul) [inline package.json:186]: Is this really necessary? "2.0.0 • Public • Published 8 years ago" seems unmaintained and it only makes HTML more readable?

File: tests/helpers/Participant.ts (hunk 2 above)
Comment (damencho) [reply tests/helpers/Participant.ts:431]: Execute is sync if I remember correctly

Comment (damencho) [reply tests/helpers/Participant.ts:431]: I will double check

Comment (saghul) [reply tests/helpers/Participant.ts:431]: If that's the case then drop the async from the function definition?

File: package.json (hunk 3 above)
Comment (saghul) [reply package.json:186]: Ah, it's a dev dependency. Sorry, nevermind!

File: tests/helpers/Participant.ts (hunk 2 above)
Comment (damencho) [reply tests/helpers/Participant.ts:431]: Ok, will do. There is another function executeAsync when you want async.

File: tests/helpers/Participant.ts (hunk 1 above)
Comment (damencho) [reply tests/helpers/Participant.ts:415]: Nope.

image

File: tests/helpers/Participant.ts (hunk 2 above)
Comment (damencho) [reply tests/helpers/Participant.ts:431]: Yep, the example is with await https://webdriver.io/docs/api/browser/execute/

**Pull request merged**